<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Artur Taranchiev</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>265</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>265</width>
    <height>480</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>265</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Passgen</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icon/key.png</normaloff>:/icon/key.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Password</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Master password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Salt</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Password length</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinPasswordLength">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Password length&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>16</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineMasterPassword">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>241</width>
       <height>26</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Master password&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineSalt">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>241</width>
       <height>26</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;For example site name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>spinPasswordLength</zorder>
    <zorder>lineSalt</zorder>
    <zorder>lineMasterPassword</zorder>
   </widget>
   <widget class="QGroupBox" name="groupMethods">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>241</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Method</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="radioShaSum">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>108</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>sha1sum</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioMd5Sum">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>108</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>md5sum</string>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="plainGeneratedPassword">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Generated password&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCopy">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy(Enter)</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>265</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>&amp;Menu</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineMasterPassword</tabstop>
  <tabstop>lineSalt</tabstop>
  <tabstop>spinPasswordLength</tabstop>
  <tabstop>radioShaSum</tabstop>
  <tabstop>radioMd5Sum</tabstop>
  <tabstop>plainGeneratedPassword</tabstop>
  <tabstop>buttonCopy</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
